apps/shooting-clubs/acceptance/features/authority-details.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const steps = require('../../');
+
+module.exports = {};
+
+Feature('Authority details step');
+
+Before((
+  I,
+  authorityDetailsPage
+) => {
+  I.visitPage(authorityDetailsPage, steps)
+});
+
+Scenario('the correct form elements are present', (
+  I,
+  authorityDetailsPage
+) => {
+  I.seeElement(authorityDetailsPage.field.referenceNumber);
+});
+
+Scenario('an error appears if I submit the form with an empty field', (
+  I,
+  authorityDetailsPage
+) => {
+  I.submitForm();
+  I.seeErrors(authorityDetailsPage.field.referenceNumberGroup);
+});
+
+Scenario('I can go to the next page if I correctly fill the form', (
+  I,
+  authorityDetailsPage,
+  clubNamePage
+) => {
+  authorityDetailsPage.fillFormAndSubmit();
```

Comment:
Very good point.  However, I did want to create a general function called fillFormAndSubmit() as we are redefiniing it so many times in our feature files.  Now the question is how do I do that for so acceptance, perhaps put it into steps.js in so-acceptance?